unicef/juniper-portfolio
# FEATURE: Admin privileges

Requires #139 

Separates Admin buttons from normal user accounts

To Test
1) Invite yourself to Juniper as a general account
2) Test in production (Build the client) or change the invite url from localhost:9000 to localhost:3000

3) Verify Wallets, Track Wallets, Startups, Donors, Natcoms, Unpublished, Published, Archived Transactions, and Settings admin functions are hidden from the general account.
